src/scripts/langs.ts

```diff
@@ -41,7 +43,7 @@ export const subsets = {
 	hu: 'latin-ext',
 	sr: 'cyrillic',
 	gr: 'greek',
-	fa: 'arabic',
+	fa: 'farsi',
```

Comment:
This is not a mistake! The subset of fonts for Arabic languages is called "arabic", and Farsi is part of the "arabic" subset.

Reply:
Done.